graylog2-web-interface/src/components/common/HoverForHelp.jsx

```diff
@@ -0,0 +1,39 @@
+// @flow strict
```

Comment:
I moved this file with `git mv`, not sure why it is being tracked as a new file.